dex-ws-load/build.sbt

```diff
@@ -0,0 +1,17 @@
+enablePlugins(GatlingPlugin)
+
+scalaVersion := "2.12.10"
```

Comment:
~~Interesting, why we can't use 2.13.2 here as in other modules in the project?~~
I've found, it is because of Gatling.